### PR TITLE
Dev prod settings

### PR DESCRIPTION
**Pull request by visse0001**
- Add settings_dev and settings_prod modules
- add ENVIRONMENT to .local.env

To run the application with a token, set local.env ENVIRONMENT to DEV, if not - set to PROD.

Thanks Leszek B for your help :)

### CODE REVIEW

File: zeton_backend/settings.py [hunk 1]
```diff
@@ -15,6 +15,11 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 from utils.conversion import str_to_boolean
 
+if os.getenv('ENVIRONMENT') and os.getenv('ENVIRONMENT') == 'DEV':
```
Comment (tgbdc7) [inline zeton_backend/settings.py]: Why do we have a double condition? `os.getenv('ENVIRONMENT')`?

File: README.md [hunk 2]
```diff
@@ -20,7 +20,9 @@ Allows you to earn points for your activities and exchange them for prizes.
 
 ## Docker and docker-compose
 
-[Install Docker](https://docs.docker.com/get-docker/)
+[Install Docker](https://docs.docker.com/get-docker/)  
+[Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
```
Comment (tgbdc7) [inline README.md]: maybe add information that this step is optional. ex `This section contains optional procedures for configuring Linux hosts to work better with Docker.`

File: env_config/local.env [hunk 3]
```diff
@@ -1 +1,2 @@
-CORS_ORIGIN_WHITELIST=localhost:3000
\ No newline at end of file
+CORS_ORIGIN_ALLOW_ALL=True
```
Comment (tgbdc7) [inline env_config/local.env:1]: Can you explain what is `CORS_ORIGIN_ALLOW_ALL`?

**Review by tgbdc7 — COMMENTED** [review]
This is working fine. I have a few questions. I wrote them in the code comments.

File: env_config/local.env (hunk 3 above)
Comment (visse0001) [reply env_config/local.env:1]: CORS is a mechanism for allowing clients to interact with APIs that are hosted on a different domain.

CORS for all domains by adding the following setting:
CORS_ORIGIN_ALLOW_ALL = True
This change isn't branch related, but I wanted to set it up like Mikołaj did.

or only enable CORS for specified domains, eg:
CORS_ORIGIN_WHITELIST=localhost:3000
Here zeton_django will allow only one domain - localhost:3000

File: zeton_backend/settings.py (hunk 1 above)
Comment (visse0001) [reply zeton_backend/settings.py]: I deleted redundant if condition. Thanks

File: README.md (hunk 2 above)
Comment (visse0001) [reply README.md]: I added more info about it. Done :)

**Review by tgbdc7 — APPROVED** [review]

**Pull request merged**